src/metabase/db/metadata_queries.clj

```diff
@@ -53,3 +53,11 @@
   [{field-id :id :as field}]
   (-> (field-query field (ql/aggregation {} (ql/count (ql/field-id field-id))))
       first first int))
+
+(defn field-max-min
+  "Returns a pair [max min] for the given `field`"
+  [{field-id :id :as field}]
+  (->> (field-query field (ql/aggregation {} [(ql/min (ql/field-id field-id))
```

Comment:
yeah there's some logic that sorts the columns when they come back from the results